time stamp
pilot/Measurement measurement
pilot/Command command
pilot/MotorInput motor_input